tests/e2e/e2e_trafficsplit_recursive_split.go

```diff
@@ -0,0 +1,323 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"sync"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	. "github.com/openservicemesh/osm/tests/framework"
+)
+
+var _ = OSMDescribe("Test traffic split where root service is same as backend service",
+	OSMDescribeInfo{
+		Tier:   1,
+		Bucket: 1,
```

Comment:
Does this need to be Tier 1 (run pre-merge)? If not, move this to Tier 2 so it doesn't increase pre-merge CI times.

Additionally, we need to bucket the test such that the buckets are more or less balanced, and update e2e/README to list this test. Currently:

![image](https://user-images.githubusercontent.com/21697719/110817670-5e35c780-8241-11eb-8f80-0dce2bd1fde4.png)

Reply:
I do think this should be tier 1 since it is testing a core feature's functionality not an add on or plugin or extra, non required feature.

Reply:
if its needed in Tier 1, lets move it to bucket 2 to balance the time

Reply:
> I do think this should be tier 1 since it is testing a core feature's functionality not an add on or plugin or extra, non required feature.

We have a lot of core features running in Tier 2 :-)
Typically, I tend to put variations of a feature (such as this one where we are testing a particular scenario of traffic split) in Tier 2.